src/GameKit/GKLeaderboardSet.cs

```diff
@@ -0,0 +1,60 @@
+//
+// GKLeaderBoardSet.cs: Non-generated API for GKLeaderBoard.
+//
+// Authors:
+//   Whitney Schmidt (whschm@microsoft.com)
+//
+// Copyright 2020, Microsoft Corp. All rights reserved.
+//
+// The class can be either constructed from a string (from user code)
+// or from a handle (from iphone-sharp.dll internal calls).  This
+// delays the creation of the actual managed string until actually
+// required
+//
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+#if !MONOMAC
+
+using System;
+using Foundation;
+using ObjCRuntime;
+using UIKit;
+
+namespace GameKit {
+	public partial class GKLeaderboardSet {
+
+		public void LoadLeaderboardsWithHandler (GKLeaderboardsHandler completionHandler)
```

Comment:
remove `WithHandler` since it's implied

Reply:
ideally there would be a `[Async]` version of the API
It will need to be manually added in this case

Reply:
Unfortunately there's already an API named `LoadLeaderboards` in `GKLeaderboardSet` namespace.

The group of related APIs have this (not great) naming scheme:

1) `LoadLeaderboards` - deprecated, generated
2) `LoadLeaderboardsWithCompletionHandler` - new! generated
3) `LoadLeaderboardsWithHandler` - new! manual wrapper for the above

So we can change the names of 2 & 3 but not 1. 

Ideas for other names very welcome!

Reply:
are the parameter names blocking the use of overloads ?
if so then I don't have any better idea atm

Reply:
Yep, they all have the same signature, `(GKLeaderboardsHandler completionHandler)` 😕